exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/configuration/ConfigurationDao.java

```diff
@@ -48,11 +50,14 @@ public ExposureConfiguration getLatestExposureConfiguration() {
                 toList(rs.getArray("days_since_last_exposure_scores")),
                 toList(rs.getArray("duration_scores")),
                 toList(rs.getArray("transmission_risk_scores")),
-                toList(rs.getArray("duration_at_attenuation_thresholds"))
+                toList(rs.getArray("duration_at_attenuation_thresholds")),
+                toList(rs.getArray("duration_at_attenuation_weights")),
+                rs.getInt("exposure_risk_duration")
```

Comment:
I still don't think this name is very descriptive, but we had that discussion elsewhere.

Reply:
Yep, it might be something else or more descriptive e.g. time unit included in name, but we decided to keep it as it is and we could add some documentation if necessary.